source/common/json/config_schemas.cc

```diff
@@ -19,7 +19,8 @@ const std::string Json::Schema::LISTENER_SCHEMA(R"EOF(
               "type" : "string"
             }
           },
-          "cipher_suites" : {"type" : "string"}
+          "cipher_suites" : {"type" : "string"},
```

Comment:
Empty string means that `SSL_CTX_set1_curves_list` won't be called and that BoringSSL is going to use its default curves (currently: `X25519:P-256:P-384`).

Having said that, setting `cipher_suites` and/or `ecdh_curves` to an empty value probably shouldn't be allowed, since the result is most likely not what user intended to do.